### ISSUE TITLE
Why do I need to repeat the `parserOptions.ecmaVersion` in my config?

### ISSUE THREAD

**Issue by edouard-lopez** [eslint/eslint#9191]
related: DockYard/eslint-plugin-ember-suave#53

**Tell us about your environment**

* **ESLint Version:**

      v3.19.0

* **Node Version:**

      v8.4.0

* **npm Version:**

      5.3.0

**What parser (default, Babel-ESLint, etc.) are you using?**

    default

**Please show your full configuration:**

<details>
<summary>Project Configuration</summary>

```js
module.exports = {
  root: true,
  extends: ['my_config/ember'],
  rules: {}
};
```
</details>

<details>
<summary><i>my_config/ember.js</i> Configuration</summary>

 ```js 
module.exports = {
  env: {
    browser: true
  },
  extends: ['./base.js', 'plugin:ember-suave/recommended']
};
```
</details>

<details>
<summary><i>my_config/base.js</i> Configuration</summary>

```js 
module.exports = {
  extends: ['eslint:recommended'],
  env: {
    es6: true
  },
  parserOptions: {
    ecmaVersion: 2017,
    sourceType: 'module'
  },
  rules: {
    // ES6
  }
};
```
</details>

<br/>

**What did you do? Please include the actual source code causing the issue.**

When I lint one of my file

    $ eslint ./tests/acceptance/login-form-test.js

**What did you expect to happen?**

I expected the  `parserOptions.ecmaVersion` from _my_config/base.js_ to be available in the project's context as per extending hierarchy:  _.eslintrc.js_ → _my_config/ember.js_ → _my_config/base.js_.

If I add `parserOptions.ecmaVersion` in the project _.eslintrc.js_ configuration file, no more error.

```js
  parserOptions: {
    ecmaVersion: 2017
  }
```

**What actually happened? Please include the actual, raw output from ESLint.**

If I remove `parserOptions.ecmaVersion` I got the error:

    /data/projects/employee-app-front/tests/acceptance/login-form-test.js
      14:61  error  Parsing error: Unexpected token function

Error is on the `function` of this line:

```js
test('clicking "forgotten…" link redirect to /reset', async function(assert) {
```

### Question

Why do I need to repeat the `parserOptions.ecmaVersion` in my config?

**Comment by not-an-aardvark** [comment]
This is occurring because you're also extending [`plugin:eslint-plugin-ember-suave/recommended`](https://github.com/DockYard/eslint-plugin-ember-suave/blob/bb7b5312e81e501f74618e4494ce1d024a618738/config/recommended.js#L4), which sets `ecmaVersion: 6`. So the setting of `parserOptions: { ecmaVersion: 8 }` in `my_config/base.js` is getting overwritten in `plugin:eslint-plugin-ember-suave/recommended`.

To address this, one option would be to change the order of the `extends` array in `my_config/ember.js` so that the `base.js` config gets applied last.

**Comment by edouard-lopez** [comment]
@not-an-aardvark my _base.js_ config extends _eslint:recommended_, will the rule be cascading in this order?
1. ember-suave
1. eslint:recommended
1. base.js' rule

**Comment by edouard-lopez** [comment]
I remove the `extends` from my _base.js_ and add the `eslint:recommended` to my _ember.js_ config:

    extends: ['eslint:recommended', 'plugin:ember-suave/recommended', './core.js']
